web/ui/react-app/src/pages/graph/GraphControls.tsx

```diff
@@ -41,6 +41,8 @@ class GraphControls extends Component<GraphControlsProps> {
     14 * 24 * 60 * 60,
     28 * 24 * 60 * 60,
     56 * 24 * 60 * 60,
+		112 * 24 * 60 * 60,
```

Comment:
The indentation on these is tabs vs. spaces in the rest of the code (probably the linter check will yell too :)).

You can run `make react-app-lint-fix` locally by the way to fix it automatically.

Reply:
Oops, I wonder how that happened.